remote-processor/RemoteProcessorServer.cpp

```diff
@@ -44,8 +44,12 @@ using std::string;
 CRemoteProcessorServer::CRemoteProcessorServer(uint16_t uiPort, IRemoteCommandHandler* pCommandHandler) :
     _uiPort(uiPort), _pCommandHandler(pCommandHandler), _bIsStarted(false), _pListeningSocket(NULL), _ulThreadId(0)
 {
+}
+
+bool CRemoteProcessorServer::init()
+{
     // Create inband pipe
-    pipe(_aiInbandPipe);
+    return pipe(_aiInbandPipe) == 0;
```

Comment:
you could do that in the "start()" method instead of adding a new method. See https://github.com/01org/parameter-framework/pull/29/files were work has started for checking the return values of libc calls.